sync.cmd

```diff
@@ -53,18 +60,28 @@ call %~dp0init-tools.cmd
 if [%src%] == [true] (
   echo Fetching git database from remote repos ...
   call git fetch --all -p -v >> %synclog% 2>&1
-  if NOT [%ERRORLEVEL%]==[0] (
+  if NOT [!ERRORLEVEL!]==[0] (
     echo ERROR: An error occurred while fetching remote source code, see %synclog% for more details.
     exit /b
   )
 )
 
+if [%azureBlobs%] == [true] (
+  echo Connecting and downloading packages from Azure BLOB ...
```

Comment:
The packages that contain the product binaries that were built and stored in Azure by the build pipeline